### DISCUSSION TITLE
Failed to load providerinstaller module: No acceptable module found. Local version is 0 and remote version is 0.

### DISCUSSION THREAD

**Discussion by getwait** [firebase/flutterfire#7928] [Q&A]
Hello, im using flutter for create my app.
I have successfully initialized firebase following the official documentation, but when i tried to get data from firestore i get this log 
`W/DynamiteModule( 2643): Local module descriptor class for providerinstaller not found.
I/DynamiteModule( 2643): Considering local module providerinstaller:0 and remote module providerinstaller:0
W/ProviderInstaller( 2643): Failed to load providerinstaller module: No acceptable module found. Local version is 0 and remote version is 0.
D/EGL_emulation( 2643): app_time_stats: avg=167.38ms min=16.65ms max=501.63ms count=6
V/NativeCrypto( 2643): Registering com/google/android/gms/org/conscrypt/NativeCrypto's 294 native methods...
W/terfire_sample( 2643): Accessing hidden method Ljava/security/spec/ECParameterSpec;->getCurveName()Ljava/lang/String; (unsupported, reflection, allowed)
I/ProviderInstaller( 2643): Installed default security provider GmsCore_OpenSSL
W/terfire_sample( 2643): Accessing hidden field Ljava/net/Socket;->impl:Ljava/net/SocketImpl; (unsupported, reflection, allowed)
W/terfire_sample( 2643): Accessing hidden method Ljava/security/spec/ECParameterSpec;->setCurveName(Ljava/lang/String;)V (unsupported, reflection, allowed)`

Can someone please help me? Thanks.

**Comment by jhaym3s** [comment]
You probably used a <TextInputType.name> 
Add the following permission to android/app/src/main/AndroidManifest.xml, before <application> starts:

    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />

**Reply by cedvdb** [reply]
why is that required ?

**Reply by uemirhanselim** [reply]
I will add this in manifest ?
When I add this before manifest, it says manifest is not valid.

**Comment by leva0212** [comment]
> Hello, im using flutter for create my app. I have successfully initialized firebase following the official documentation, but when i tried to get data from firestore i get this log `W/DynamiteModule( 2643): Local module descriptor class for providerinstaller not found. I/DynamiteModule( 2643): Considering local module providerinstaller:0 and remote module providerinstaller:0 W/ProviderInstaller( 2643): Failed to load providerinstaller module: No acceptable module found. Local version is 0 and remote version is 0. D/EGL_emulation( 2643): app_time_stats: avg=167.38ms min=16.65ms max=501.63ms count=6 V/NativeCrypto( 2643): Registering com/google/android/gms/org/conscrypt/NativeCrypto's 294 native methods... W/terfire_sample( 2643): Accessing hidden method Ljava/security/spec/ECParameterSpec;->getCurveName()Ljava/lang/String; (unsupported, reflection, allowed) I/ProviderInstaller( 2643): Installed default security provider GmsCore_OpenSSL W/terfire_sample( 2643): Accessing hidden field Ljava/net/Socket;->impl:Ljava/net/SocketImpl; (unsupported, reflection, allowed) W/terfire_sample( 2643): Accessing hidden method Ljava/security/spec/ECParameterSpec;->setCurveName(Ljava/lang/String;)V (unsupported, reflection, allowed)`
> 
> Can someone please help me? Thanks.

add the next dependenci line to the build.gradle file in app folder
implementation("com.squareup.okhttp3:okhttp:4.9.3")
this will fix the problem, then the application will by able to conect to firebase correctly

**Reply by darshan9954** [reply]
Oooo.. good

**Reply by uemirhanselim** [reply]
This didn't work for me

**Reply by wilerbriones** [reply]
You`re the best....!!!! I was trying for more then 20 hours, until finnaly I found your suggestion. It really worked!!!!! Thank you.

**Reply by simonka0808** [reply]
I added this line of code but still getting that error. Can you please help me, i ve been trying to fix this for weeks

**Reply by EmanuelBalaban** [reply]
Still having this issue in 2024.

Adding implementation("com.squareup.okhttp3:okhttp:4.9.3") to build.gradle didn't solve it for me.

**Comment by uemirhanselim** [comment]
Did anyone find a solution, I am still stuck at this?

**Reply by NBMinh63343** [reply]
me too

**Reply by uemirhanselim** [reply]
I solved btw. Turning off then turning on wifi worked for me. (Emulator's wifi)

**Reply by osder89** [reply]
> Resolví por cierto. Apagar y luego encender wifi funcionó para mí. (Wifi del emulador)

Funciona en mi caso funciono apagando el emulador y volviendo a darle los permisos para el wifi

**Comment by t31v12d98** [comment]
Pls Help me :(
`W/DynamiteModule( 2768): Local module descriptor class for com.google.android.gms.providerinstaller.dynamite not found.
I/DynamiteModule( 2768): Considering local module com.google.android.gms.providerinstaller.dynamite:0 and remote module com.google.android.gms.providerinstaller.dynamite:0
W/ProviderInstaller( 2768): Failed to load providerinstaller module: No acceptable module com.google.android.gms.providerinstaller.dynamite found. Local version is 0 and remote version is 0.
I/om.example.qbo( 2768): The ClassLoaderContext is a special shared library.
I/chatty  ( 2768): uid=10097(com.example.qbox) AsyncTask #2 identical 1 line
I/om.example.qbo( 2768): The ClassLoaderContext is a special shared library.
V/NativeCrypto( 2768): Registering com/google/android/gms/org/conscrypt/NativeCrypto's 295 native methods...
W/om.example.qbo( 2768): Accessing hidden method Ljava/security/spec/ECParameterSpec;->getCurveName()Ljava/lang/String; (light greylist, reflection)`

**Comment by seung1** [comment]
Hi guys
First of all, I am Korean, please understand that my English is not good.
I also had the same problem. Now I solved it.

The problem occurs in the firebase console.

1. Go to the firebase site.
2. Select your firebase project.
3. Click Firestore Database on the left menu.
4. Click Rules in the top menu.
5. Maybe your rules are

```
rules_version = '2';
service cloud. firestore {
   match /databases/{database}/documents {
     match /{document=**} {
       allow read, write: if false;
     }
   }
}
```

6. If you change "false" to "true" here, the problem will be solved.

Here's how to avoid causing problems:
When creating a database, you have two options.
1. Start in production mode
2. Start in test mode

If you choose the second here, the problem will not arise.

----------------------------------------------------------------
혹시 해당 문제가 생긴 한국인이 있다면
데이터베이스 생성하실때 선택지가 2개있는데  (프로덕션 모드 / 테스트 모드)
프로덕션 모드를 선택하게 되면 발생하는 문제같습니다.
저도 배우는 입장이라 정확히는 모르지만

프로덕션 모드로 하면 기본적으로 읽기쓰기가 비공개로 설정돼서
기본적인 방법으로는 읽고쓰기가 안되는거 같아요
아마 보안설정인듯싶습니다.
테스트 모드로해도 30일정도만 가능한거같습니다.
더 정확히 아시는분은 댓글로 써주세요 

해결책은 2가지 있습니다.
1. 다시 지웠다가 다시만들면서 테스트 모드로 만들기
2. 해당 프로젝트 들어가셔서 Firestore Database 누르시고 상단에 규칙누르신 다음에 false부분을 수정하시면 됩니다. true로 하셔도 가능은 한데 기본 테스트모드 코드는 아래와 같습니다.

```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{document=**} {
      allow read, write: if
          request.time < timestamp.date(2022, 12, 17);
    }
  }
}
```

위에코드는 22년 12월 17일까지 가능하다는건데 제가 지금이거 작성하는 날로부터 30일이후 날짜입니다.

**Comment by mcbryan1** [comment]
I Killed and started the server again. That's what worked for me

**Reply by tahakockar** [reply]
Killed yapmak ne oluyor ?

**Comment by AqibAli888** [comment]
I have also that issue and i try lots of searches to find better solution and at  the end i found that if you fetch data with "async" and "await" then there will be no error Thank you!

**Comment by miran76543** [comment]
Plesss help me my map do crash 




`E/AndroidRuntime(13627): Process: com.example.miranluqman, PID: 13627
E/AndroidRuntime(13627): java.lang.IncompatibleClassChangeError: Found interface com.google.android.gms.location.FusedLocationProviderClient, but class was expected (declaration of 'com.google.android.gms.location.FusedLocationProviderClient' appears in /data/app/com.example.miranluqman-1UI4GR-yEN8Sln-kagMYpg==/base.apk)E/AndroidRuntime(13627):        at com.lyokone.location.FlutterLocation.createLocationCallback(FlutterLocation.java:219)
E/AndroidRuntime(13627):        at com.lyokone.location.FlutterLocation.changeSettings(FlutterLocation.java:197)
E/AndroidRuntime(13627):        at com.lyokone.location.MethodCallHandlerImpl.onChangeSettings(MethodCallHandlerImpl.java:106)
E/AndroidRuntime(13627):        at com.lyokone.location.MethodCallHandlerImpl.onMethodCall(MethodCallHandlerImpl.java:40)
E/AndroidRuntime(13627):        at io.flutter.plugin.common.MethodChannel$IncomingMethodCallHandler.onMessage(MethodChannel.java:258)
E/AndroidRuntime(13627):        at io.flutter.embedding.engine.dart.DartMessenger.invokeHandler(DartMessenger.java:295)
DartMessenger.java:322)
E/AndroidRuntime(13627):        at io.flutter.embedding.engine.dart.DartMessenger$$ExternalSyntheticLambda0.run(Unknown Source:12)
E/AndroidRuntime(13627):        at android.os.Handler.handleCallback(Handler.java:907)
E/AndroidRuntime(13627):        at android.os.Handler.dispatchMessage(Handler.java:105)
E/AndroidRuntime(13627):        at android.os.Looper.loop(Looper.java:216)
E/AndroidRuntime(13627):        at android.app.ActivityThread.main(ActivityThread.java:7625)
E/AndroidRuntime(13627):        at java.lang.reflect.Method.invoke(Native Method)
E/AndroidRuntime(13627):        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:524)
E/AndroidRuntime(13627):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:987)
I/Process (13627): Sending signal. PID: 13627 SIG: 9
Lost connection to device.`

**Comment by AbhiCodebugger** [comment]
Launching lib\main.dart on sdk gphone64 x86 64 in debug mode...
√  Built build\app\outputs\flutter-apk\app-debug.apk.
Connecting to VM Service at ws://127.0.0.1:57798/Anh1e0Ioa3k=/ws
D/CompatibilityChangeReporter(15171): Compat change id reported: 183155436; UID 10194; state: ENABLED
I/FirebaseApp(15171): Device unlocked: initializing all Firebase APIs for app [DEFAULT]
W/DynamiteModule(15171): Local module descriptor class for com.google.android.gms.providerinstaller.dynamite not found.
I/DynamiteModule(15171): Considering local module com.google.android.gms.providerinstaller.dynamite:0 and remote module com.google.android.gms.providerinstaller.dynamite:0
W/ProviderInstaller(15171): Failed to load providerinstaller module: No acceptable module com.google.android.gms.providerinstaller.dynamite found. Local version is 0 and remote version is 0.
D/nativeloader(15171): Configuring clns-7 for other apk /system/framework/org.apache.http.legacy.jar. target_sdk_version=33, uses_libraries=ALL, library_path=/product/priv-app/PrebuiltGmsCore/lib/x86_64:/product/priv-app/PrebuiltGmsCore/PrebuiltGmsCore.apk!/lib/x86_64, permitted_path=/data:/mnt/expand:/data/user/0/com.google.android.gms
D/nativeloader(15171): Configuring clns-8 for other apk /system/framework/com.android.media.remotedisplay.jar. target_sdk_version=33, uses_libraries=ALL, library_path=/product/priv-app/PrebuiltGmsCore/lib/x86_64:/product/priv-app/PrebuiltGmsCore/PrebuiltGmsCore.apk!/lib/x86_64, permitted_path=/data:/mnt/expand:/data/user/0/com.google.android.gms
D/nativeloader(15171): Configuring clns-9 for other apk /system/framework/com.android.location.provider.jar. target_sdk_version=33, uses_libraries=ALL, library_path=/product/priv-app/PrebuiltGmsCore/lib/x86_64:/product/priv-app/PrebuiltGmsCore/PrebuiltGmsCore.apk!/lib/x86_64, permitted_path=/data:/mnt/expand:/data/user/0/com.google.android.gms
D/ApplicationLoaders(15171): Returning zygote-cached class loader: /system_ext/framework/androidx.window.extensions.jar
D/ApplicationLoaders(15171): Returning zygote-cached class loader: /system_ext/framework/androidx.window.sidecar.jar
D/nativeloader(15171): InitLlndkLibrariesProduct: libEGL.so:libGLESv1_CM.so:libGLESv2.so:libGLESv3.so:libRS.so:libandroid_net.so:libbinder_ndk.so:libc.so:libcgrouprc.so:libclang_rt.asan-x86_64-android.so:libcom.android.tethering.connectivity_native.so:libdl.so:libft2.so:liblog.so:libm.so:libmediandk.so:libnativewindow.so:libneuralnetworks.so:libselinux.so:libsync.so:libvndksupport.so:libvulkan.so
D/nativeloader(15171): Configuring product-clns-10 for unbundled product apk /product/priv-app/PrebuiltGmsCore/PrebuiltGmsCore.apk. target_sdk_version=33, uses_libraries=, library_path=/product/priv-app/PrebuiltGmsCore/lib/x86_64:/product/priv-app/PrebuiltGmsCore/PrebuiltGmsCore.apk!/lib/x86_64:/product/lib64:/system/product/lib64, permitted_path=/data:/mnt/expand:/data/user/0/com.google.android.gms:/product/lib64:/system/product/lib64
D/nativeloader(15171): InitVndkspLibrariesProduct: android.hardware.common-V2-ndk.so:android.hardware.common.fmq-V1-ndk.so:android.hardware.graphics.allocator-V2-ndk.so:android.hardware.graphics.common-V4-ndk.so:android.hardware.graphics.common@1.0.so:android.hardware.graphics.common@1.1.so:android.hardware.graphics.common@1.2.so:android.hardware.graphics.composer3-V1-ndk.so:android.hardware.graphics.mapper@2.0.so:android.hardware.graphics.mapper@2.1.so:android.hardware.graphics.mapper@3.0.so:android.hardware.graphics.mapper@4.0.so:android.hardware.renderscript@1.0.so:android.hidl.memory.token@1.0.so:android.hidl.memory@1.0-impl.so:android.hidl.memory@1.0.so:android.hidl.safe_union@1.0.so:libRSCpuRef.so:libRSDriver.so:libRS_internal.so:libbase.so:libbcinfo.so:libblas.so:libc++.so:libcompiler_rt.so:libcutils.so:libdmabufheap.so:libgralloctypes.so:libhardware.so:libhidlbase.so:libhidlmemory.so:libion.so:libjsoncpp.so:liblzma.so:libprocessgroup.so:libunwindstack.so:libutils.so:libutilscallstack.so:libz.so
V/NativeCrypto(15171): Registering com/google/android/gms/org/conscrypt/NativeCrypto's 295 native methods...
W/example.my_todo(15171): Accessing hidden method Ljava/security/spec/ECParameterSpec;->getCurveName()Ljava/lang/String; (unsupported, reflection, allowed)
I/ProviderInstaller(15171): Installed default security provider GmsCore_OpenSSL

This is the place it gets stuck and never loads.
HELP !!

**Comment by henri045** [comment]
════════ Exception caught by image resource service ════════════════════════════
The following ArgumentError was thrown resolving an image codec:
Invalid argument(s): No host specified in URI file:///

When the exception was thrown, this was the stack:
#0      _HttpClient._openUrl (dart:_http/http_impl.dart:2752:9)
#1      _HttpClient.getUrl (dart:_http/http_impl.dart:2628:48)
#2      NetworkImage._loadAsync (package:flutter/src/painting/_network_image_io.dart:104:59)
#3      NetworkImage.loadImage (package:flutter/src/painting/_network_image_io.dart:66:14)
#4      ImageProvider.resolveStreamForKey.<anonymous closure> (package:flutter/src/painting/image_provider.dart:520:39)
#5      ImageCache.putIfAbsent (package:flutter/src/painting/image_cache.dart:377:22)
#6      ImageProvider.resolveStreamForKey (package:flutter/src/painting/image_provider.dart:517:81)
#7      ScrollAwareImageProvider.resolveStreamForKey (package:flutter/src/widgets/scroll_aware_image_provider.dart:104:19)
#8      ImageProvider.resolve.<anonymous closure> (package:flutter/src/painting/image_provider.dart:366:9)
#9      ImageProvider._createErrorHandlerAndKey.<anonymous closure> (package:flutter/src/painting/image_provider.dart:479:24)
#10     SynchronousFuture.then (package:flutter/src/foundation/synchronous_future.dart:43:39)
#11     ImageProvider._createErrorHandlerAndKey (package:flutter/src/painting/image_provider.dart:476:9)
#12     ImageProvider.resolve (package:flutter/src/painting/image_provider.dart:363:5)
#13     _ImageState._resolveImage (package:flutter/src/widgets/image.dart:1110:16)
#14     _ImageState.didChangeDependencies (package:flutter/src/widgets/image.dart:1060:5)
#15     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5620:11)
#16     ComponentElement.mount (package:flutter/src/widgets/framework.dart:5447:5)
...     Normal element mounting (74 frames)
#90     Element.inflateWidget (package:flutter/src/widgets/framework.dart:4326:16)
#91     Element.updateChild (package:flutter/src/widgets/framework.dart:3837:18)
#92     SlottedRenderObjectElement._updateChildren (package:flutter/src/widgets/slotted_render_object_widget.dart:295:33)
#93     SlottedRenderObjectElement.mount (package:flutter/src/widgets/slotted_render_object_widget.dart:249:5)
...     Normal element mounting (186 frames)
#279    Element.inflateWidget (package:flutter/src/widgets/framework.dart:4326:16)
#280    Element.updateChild (package:flutter/src/widgets/framework.dart:3837:18)
#281    SliverMultiBoxAdaptorElement.updateChild (package:flutter/src/widgets/sliver.dart:858:37)
#282    SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:843:20)
#283    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2835:19)
#284    SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:835:12)
#285    RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:349:23)
#286    RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:2657:59)
#287    PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:1071:15)
#288    RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:2657:14)
#289    RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:338:5)
#290    RenderSliverMultiBoxAdaptor.insertAndLayoutChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:486:5)
#291    RenderSliverList.performLayout.advance (package:flutter/src/rendering/sliver_list.dart:239:19)
#292    RenderSliverList.performLayout (package:flutter/src/rendering/sliver_list.dart:281:12)
#293    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#294    RenderSliverEdgeInsetsPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:139:12)
#295    RenderSliverPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:361:11)
#296    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#297    RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:601:13)
#298    RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1554:12)
#299    RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:1463:20)
#300    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#301    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#302    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#303    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#304    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#305    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#306    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#307    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#308    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#309    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#310    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#311    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#312    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#313    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#314    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#315    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#316    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#317    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#318    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#319    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#320    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#321    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#322    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#323    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:104:21)
#324    _RenderCustomClip.performLayout (package:flutter/src/rendering/proxy_box.dart:1434:11)
#325    RenderObject.layout (package:flutter/src/rendering/object.dart:2546:7)
#326    RenderBox.layout (package:flutter/src/rendering/box.dart:2389:11)
#327    MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:173:12)
#328    _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:1062:7)
#329    MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:237:7)
#330    RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:403:14)
#331    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:2385:7)
#332    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:1025:18)
#333    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:1038:15)
#334    RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:591:23)
#335    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:986:13)
#336    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:457:5)
#337    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1325:15)
#338    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1255:9)
#339    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:1113:5)
#340    _invoke (dart:ui/hooks.dart:312:13)
#341    PlatformDispatcher._drawFrame (dart:ui/platform_dispatcher.dart:383:5)
#342    _drawFrame (dart:ui/hooks.dart:283:31)

Image provider: NetworkImage("", scale: 1.0)
Image key: NetworkImage("", scale: 1.0)
════════════════════════════════════════════════════════════════════════════════
Restarted application in 1,126ms.
W/DynamiteModule( 5604): Local module descriptor class for com.google.android.gms.providerinstaller.dynamite not found.
I/DynamiteModule( 5604): Considering local module com.google.android.gms.providerinstaller.dynamite:0 and remote module com.google.android.gms.providerinstaller.dynamite:0
W/ProviderInstaller( 5604): Failed to load providerinstaller module: No acceptable module com.google.android.gms.providerinstaller.dynamite found. Local version is 0 and remote version is 0.
D/TrafficStats( 5604): tagSocket(105) with statsTag=0xffffffff, statsUid=-1
D/EGL_emulation( 5604): app_time_stats: avg=30011.84ms min=30011.84ms max=30011.84ms count=1


I  need some help please

**Reply by alfonsocanor** [reply]
Exact same problem. I'm running android-arm64 emulator. This doens't stop me to use the app but I guess eventually it will

**Comment by kritsimar8** [comment]
Make sure you have enabled rules for read and write in cloud storage.
